build/hellobuild.md

```diff
@@ -0,0 +1,136 @@
+---
+title: Hello Build
+description: Build Hello World
+keywords: build, buildx, buildkit, getting started, Dockerfile, image layers, build instructions, build context
+---
+
+## Hello Build!
+
+It all starts with a Dockerfile.
+
+Dockerfiles are text files containing instructions. Dockerfiles adhere to a specific format and contain a **set of instructions** for which you can find a full reference in the [Dockerfile reference](../engine/reference/builder).
+<br/>Docker builds images by reading the instructions from a Dockerfile.
+
+Docker images consist of **read-only layers**, each resulting from an instruction in the Dockerfile. Layers are stacked sequentially and each one is a delta representing the changes applied to the previous layer.
+
+## Dockerfile basics
+
+A Dockerfile is a text file containing all necessary instructions needed to assemble and deploy a container image with Docker.
+
+Here are the most common types of instructions:
+ 
+* [**FROM \<image\>**](../engine/reference/builder/#from) - defines a base for your image.
+* [**RUN \<command\>**](../engine/reference/builder/#run) - executes any commands in a new layer on top of the current image and commits the result. <br/>RUN also has a shell form for running commands.
+* [**WORKDIR \<directory\>**](../engine/reference/builder/#workdir) - sets the working directory for any RUN, CMD, ENTRYPOINT, COPY and ADD instructions that follow it in the Dockerfile.
+* [**COPY \<src\> \<dest\>**](../engine/reference/builder/#copy) - copies new files or directories from \<src\> and adds them to the filesystem of the container at the path \<dest\>.
+* [**CMD \<command\>**](../engine/reference/builder/#cmd) - lets you define the default program that is run once you start the container based on this image. Each Dockerfile only has one CMD, and only the last CMD instance is respected when multiple exist.
+<br/>Dockerfiles are crucial inputs for image builds and can facilitate automated, multi-layer image builds based on your unique configurations. Dockerfiles can start simple and grow with your needs and support images that require complex instructions.
+  
+For all the possible instructions, see the [Dockerfile reference](../engine/reference/builder/).
+
+## Example
+Here’s a simple Dockerfile example to get you started with building images. We’ll take a simple "Hello World" Python Flask application, and bundle it into a docker image that we can test locally or deploy anywhere!
+
+**Sample A**
+<br/>Let’s say we have the following in a `hello.py` file in our local directory:
+
+{% highlight python lineos %}
```

Comment:
Do we need another highlight format? We've previously used backticks and specified the language (```python) . CC @thaJeztah

Reply:
`{% highlight python lineos %}` we can just use ```python yes.

This one was to try to add line numbers but it does not work atm.